lib/model/model.go

```diff
@@ -1106,15 +1074,10 @@ func (m *Model) handleIntroductions(introducerCfg config.DeviceConfiguration, cm
 			if _, ok := m.cfg.Devices()[device.ID]; !ok {
 				// The device is currently unknown. Add it to the config.
 				m.introduceDevice(device, introducerCfg)
-				changed = true
```

Comment:
This was lost somehow?

Reply:
Not lost, removed. It is superfluous as will already be set to true here:
https://github.com/syncthing/syncthing/pull/4974/files/dfa19b40a8c03563a50dd7278f6b5114768e35aa#diff-1df9805b2381b6288b220226f82d5dc9R1090

Reply:
That doesn't show anything meaningful.

Reply:
ah it's at the bottom, ok.

Reply:
I seriously need to stop linking code and start copying it - these github-diff-links are so broken.